lib/spack/spack/hooks/sbang.py

```diff
@@ -24,7 +24,7 @@
 
 def sbang_install_path():
     """Location sbang should be installed within Spack's ``install_tree``."""
-    return os.path.join(spack.store.layout.root, "bin", "sbang")
+    return os.path.join(spack.store.store.sbang_root, "bin", "sbang")
```

Comment:
It's weird that sbang has to ask the store where the sbang root is. I think this needs to be decoupled.

Reply:
Renamed to `unpadded_root`